bolt.gemspec

```diff
@@ -19,8 +19,10 @@ Gem::Specification.new do |spec|
                        Dir['lib/**/*.rb'] +
                        Dir['lib/**/*.json'] +
                        Dir['libexec/*'] +
+                       Dir['bolt-modules/*/metadata.json'] +
                        Dir['bolt-modules/*/lib/**/*.rb'] +
                        Dir['bolt-modules/*/types/**/*.pp'] +
+                       Dir['modules/*/metadata.json'] +
```

Comment:
I think we should merge this change in separately since it's obviously useful.

Reply:
Moved this one over to PR #861.